src/css/_right-side.scss

```diff
@@ -1,6 +1,7 @@
 .right-sidebar {
   grid-area: right;
-  height: calc(100vh- 28px);
+  height: auto;
+  // height: calc(100vh- 28px);
```

Comment:
If this looks better I am all for it. I was just trying to use some of the sass calculation functionality